sdk/react/widgets/ToggleSwitch.js

```diff
@@ -24,7 +25,18 @@ var ToggleSwitch = React.createClass({
     switchOnText: React.PropTypes.string,
     switchOffText: React.PropTypes.string,
     onValueChanged: React.PropTypes.func,
-    config: React.PropTypes.object
+    config: React.PropTypes.object,
+    onTintColor: React.PropTypes.string,
+    tintColor: React.PropTypes.string,
+    thumbTintColor: React.PropTypes.string,
+  },
+
+  getDefaultProps: function() {
+    return {
+      onTintColor: '#498DFC',
```

Comment:
Oh, my bad. As a user of the ToggleSwitch, you can specify the color using those props. For example:

`<ToggleSwitch 
  onTintColor={'blue'}
  tintColor={'grey'}
  thumbTintColor={'white'}  />`

That would override the default props that I'm setting in the getDefaultProps function. Would you like me to add the color props where we are using the ToggleSwitch in the LanguageSelectionPanel.js?